frontend/src/pages/CardPage/components/Slider/swiper.css

```diff
@@ -0,0 +1,8 @@
+.swiper-pagination-bullet-active {

```

Comment:
Do we need to create separate swiper.css file? Can`t we use all styles in Slider.module.scss?

Reply:
Nope, we can't. Slider.module.scss changes the class name. And i need to specify styles for swiper library classes.